reply-letter.tex

```diff
@@ -146,68 +161,71 @@ \section*{Reviewer \# 1}
-\newl
-Details.}
+Concerning the scientific results, the paper is sound and precise. The claims are supported by several numerical simulations that nicely highlight the main points of this paper. So, overall, this paper is very good, but requires reworking several sections to improve readability, reduce technical slang, and include the so far ignored scientific work that appears to be very closely related to this submitted paper.}
+
+\REPLY{We appreciate the reviewer's overall positive take on the results, and have attempted to address the important considerations regarding clarity and past work in our responses to each of the details provided below.}
 
-\REVIEW{a)      In the introduction, the work related to the gamma model should be discussed.  The gamma model is a rather complete framework that highlights the flexibility of functional mapping in terms of input/output mapping based on delayed responses, implemented by delayed filter kernels. I propose including an in depth discussion and highlighting how the new approach differs from the gamma model. (`The gamma model-A new neural model for temporal processing' by Bert de Vries and Jose C. Principe, University of Florida, USA, ScienceDirect: http://www.sciencedirect.com/science/article/pii/S0893608005800358).}
+\REVIEW{Details.
+a)      In the introduction, the work related to the gamma model should be discussed.  The gamma model is a rather complete framework that highlights the flexibility of functional mapping in terms of input/output mapping based on delayed responses, implemented by delayed filter kernels. I propose including an in depth discussion and highlighting how the new approach differs from the gamma model. (`The gamma model-A new neural model for temporal processing' by Bert de Vries and Jose C. Principe, University of Florida, USA, ScienceDirect: http://www.sciencedirect.com/science/article/pii/S0893608005800358).}
 
-\REPLY{2.1. The gamma model is related to our work in several ways, but is also quite distinct in some crucial ways that we will discuss now. 
-First, spiking neurons are not considered at all in the paper by Bert de Vries and Jose C. Principe or in any of the literature we found in section~2.
-Second, their use of the term ``delay kernel'' is referring to the fact that the output of a temporal convolution at time $t$ depends on the input at time $t - s$ (e.g.,~equation~8), as specified by some kernel of weights.
-In particular, they refer to equation~29 as a ``delay structure'' despite it being a set of differential equations implementing their gamma kernel.
-This is more appropriately referred to in the signal processing literature as a particular kind of lowpass filter (more on this below).
+\REPLY{2.1. First, spiking neurons are not considered in the paper by Bert de Vries and Jose C. Principe or in the literature we found in section~2 of VP.
```

Comment:
We agree that the gamma model is closely related and have significantly extended the introduction to incorporate a discussion highlighting similarities and differences.  We begin by highlighting some of the important differences.